wagtail/contrib/settings/templatetags/wagtailsettings_tags.py

```diff
@@ -0,0 +1,21 @@
+from django.template import Library
+
+from wagtail.wagtailcore.models import Site
+
+from ..context_processors import SettingsProxy
+
+register = Library()
+
+
+@register.simple_tag(takes_context=True)
+def get_settings(context, use_default_site=False):
```

Comment:
ok, that's fine by me.

Just note that when the "as" syntax arrives, this tag will fill the specified variable with a blank string which may not be what the developer expects.